src/index.html

```diff
@@ -1,14 +1,41 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta
+        name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
-</html>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="external" id="qa-block">
+      <div class="middle">
+        <div class="interior">
+          In programming languages, a closure is a technique for implementing
+          lexically scoped name binding in a language with first-class functions.
+          Operationally, a closure is a record storing a function together with an environment.
+        </div>
+      </div>
+    </div>
+    <div class="external">
+      <div class="middle">
+        <div class="interior">
+          У програмуванні замиканням називають підпрограму, що виконується
+          в середовищі, яке містить одну або більше зв’язаних змінних.
+          Підпрограма має доступ до цих змінних під час виконання.
+        </div>
+      </div>
+    </div>
+    <div class="external">
```

Comment:
```suggestion
    <div class="external-block">
```
external is an adjective, you need to add something this adjective describes (in your case this is block)

Reply:
changed